### PR TITLE
Fix event loading / Rich Text Forms

### LINKED ISSUE #107: Rich text event descriptions
Execs may want to add line breaks, emojis, bold font to event descriptions. We should support at least line breaks

### LINKED ISSUE #125: edit event UI fix on longitude and latitude
When editing an event and selecting from common event locations, there's a visual bug for longitude and latitude where the label will overlap with the value

### PR DESCRIPTION
**Pull request by ianmah**
image

because i removed the api call in router, some pages need to update the events manually

also fix form error in edit event

closes #125
closes #107

### CODE REVIEW

**Comment by dchen150** [comment]
does updateEvents() populate redux with events? if so wouldn't it be the same as what we had originally (having events in redux)?

**Comment by ianmah** [comment]
> does updateEvents() populate redux with events? if so wouldn't it be the same as what we had originally (having events in redux)?

Yes. Events are still in redux, they just aren't updated by the Router anymore (which is a top level component and was being called unnecessarily)

File: src/pages/admin/EventEdit.js [hunk 1]
```diff
@@ -29,18 +33,22 @@ const useStyles = makeStyles(theme => ({
...
 
-    useEffect(() => {
-        // Get the initial values
-        if (!event && events && eventId) {
-            setEvent(events.find(event => event.id === eventId));
-        }
-    }, [event, events, setEvent, eventId])
+    // Get the initial values
+    if (!event && events && eventId) {
+        let event = events.find(event => event.id === eventId)
```
Comment (dchen150) [inline src/pages/admin/EventEdit.js:48]: would it make sense to set this using `const` instead of `let`

Comment (ianmah) [reply src/pages/admin/EventEdit.js:48]: yea

**Pull request merged**